.github/workflows/build-and-push-pms-spec.yml

```diff
@@ -0,0 +1,56 @@
+name: Build and Push Plex Media Server API Spec
+
+permissions:
+  checks: write
+  contents: write
+  pull-requests: write
+  statuses: write
+
+on:
+  workflow_dispatch:
+    inputs:
+      force:
+        description: Force generation of SDKs
+        type: boolean
+        default: false
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/build-and-push-pms-spec.yaml
+      - src/**
+
+jobs:
+  build:
+    name: Build Open API Spec
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Project Setup
+        uses: ./.github/workflows/steps/project-setup
+
+      - name: Dereference Plex Media Server Specification
+        run: bun run build
+
+      - name: Commit Dereferenced Specification File
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: "build: dereferenced Plex Media Server API Spec updated"
+          skip_checkout: true
+          skip_fetch: true
+          add_options: "-f"
+          file_pattern: "./output/*.yaml"
+          skip_dirty_check: true
+  generate:
+    needs: build
+    name: Push to Speakeasy API
+    uses: speakeasy-api/sdk-generation-action/.github/workflows/workflow-executor.yaml@v15
+    with:
+      force: ${{ github.event.inputs.force }}
```

Comment:
@LukeHagar Can this be removed if this pipeline will never generate an SDK directly?